public/about.html

```
@@ -5,190 +5,108 @@
+                <h3>Support This Work</h3>
+                <p>If you enjoy this site consider supporting its continuation. We are grateful to anyone who contributes.</p>
+            </div>
+
+            <div class="crypto-support">
+                <div class="crypto-label">
+                    <span class="crypto-icon">⚡</span>
+                    <span>Zcash</span>
+                </div>
+                <div class="address-container">
+                    <code class="crypto-address" id="zcash-address">u17h46njzzm0gal5tplz8qtq009tqyxtvqu7c25uvun46nmrkux7u9064jgdk8lm8schulpz5t3rjpj7hd3rwcdsp4f86l9lxrsdxggqjffp46ry52z0hqg4g5mjvag8k4z6fkvakmtq5222qdsx20lu70l03xyn322tvz5qzeqygljjc7</code>
+                    <button class="copy-btn" onclick="copyZcashAddress()" aria-label="Copy Zcash address">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)